modules/wri_block/config/install/views.view.content_blocks_browser.yml

```diff
@@ -2,6 +2,8 @@ langcode: en
 status: true
 dependencies:
   config:
+    - block_content.type.in_this_series
+    - block_content.type.upcoming_event
```

Comment:
I had to do something special for this dependency section to avoid install dependency contradictions. Check out `wri_in_this_series_install` and do the same thing in the `wri_event_callout` install hook then leave these additions out please!